code/States/BaseState.cs

```diff
@@ -9,7 +10,7 @@ public abstract partial class BaseState : BaseNetworkable
 		public virtual int StateDuration => 0;
```

Comment:
Could be beneficial to have a comment here stating that this is in seconds (I'm assuming it is) just to avoid confusion

Reply:
Good point, I'll rename this integer.